instance/static/js/src/instance.js

```diff
@@ -69,11 +69,14 @@ app.factory('OpenCraftAPI', function(Restangular) {
     });
 });
 
+app.factory('WebSocketClient', function() {
+    return new WebSocket('ws://' + window.location.host + '/ws/');
```

Comment:
This is a problem in production where we will be using the `wss` protocol. I have to come up with a way to select the correct protocol in production and development environments.

Reply:
Excellent!